index.bs

```diff
@@ -2102,7 +2102,7 @@ and will be removed in the future versions.
 <div algorithm="remote end steps for session.unsubscribe">
 The [=remote end steps=] with |session| and |command parameters| are:
 
-1. If |command parameters| matches the <code>session.UnsubscribeByAttributesRequest</code> production:
+1. If |command parameters| does not [=map/contain=] "<code>subscriptions</code>":
```

Comment:
Quick question, here it's implied that `command parameters` already matches either `session.UnsubscribeByAttributesRequest` or `session.UnsubscribeByIDRequest`, and if `subscriptions` is not in `parameters`, then it has to match `session.UnsubscribeByAttributesRequest`. I think it makes sense overall but it's a bit hard to follow and might be worth adding a NOTE if we don't want to explicitly call out `and matches session.UnsubscribeByAttributesRequest` (which I agree is redundant)

Reply:
Added a note.